llvm/test/Transforms/LoopVectorize/vectorize-vp-intrinsics.ll

```diff
@@ -0,0 +1,98 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
```

Comment:
should this pass `-force-vector-width`? at the moment we vectorize with VF = vscale x 1 and I think this is causing a UBSan failure the running the following tests

```
 LLVM :: Transforms/LoopVectorize/vectorize-vp-intrinsics-iv32.ll
  LLVM :: Transforms/LoopVectorize/vectorize-vp-intrinsics.ll
  LLVM :: Transforms/LoopVectorize/vplan-vp-intrinsics.ll
```